numpy/lib/type_check.py

```diff
@@ -395,19 +395,23 @@ def nan_to_num(x, copy=True, nan=0.0, posinf=None, neginf=None):
         in-place (False). The in-place operation only occurs if
         casting to an array does not require a copy.
         Default is True.
+        .. versionadded:: 1.13
```

Comment:
Hmm, you need blank lines before these. Wonder why circleCI is not complaining? @mattip.

Reply:
The directive is interpreted as text, with no special formatting. See the [rendered result](https://8640-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/reference/generated/numpy.nan_to_num.html?highlight=nan_to_num#numpy.nan_to_num)

Reply:
I wasn't sure, thanks for pointing out @charris